tests/testthat/test_models.R

```diff
@@ -0,0 +1,121 @@
+
+context(desc = "Testing that canonical models work as they should")
+
+dags <- c("X -> Y", "X -> M -> Y", "X -> Y; Z -> Y")
+
+for(dag in dags){
+
+	testthat::test_that(
+		desc = "Model returns a non-null object.",
+		code = expect_true(length(make_model(dag)) > 1)
+	)
+
+	testthat::test_that(
+		desc = "Model returns the correct object class.",
+		code = expect_identical(class(model), "probabilistic_causal_model")
+	)
+
+}
+
+testthat::test_that(
+	desc = "Test functions on model X -> Y without confounding",
+	code = {
+
+		XY_noconf <- make_model("X -> Y")
+		expect_identical(dim(get_parameter_matrix(XY_noconf)), c(6,8))
```

Comment:
use `expect_equal` instead

`dim(get_parameter_matrix(XY_noconf))` returns an object of type numeric while  `c(6,8)` is integer